test/cpp/jit/test_gpu.cpp

```diff
@@ -18148,6 +18148,135 @@ TEST(NVFuserTest, FusionRfactorContigIDs_CUDA) {
   testValidate(&fusion, outputs, aten_inputs, {ref}, __LINE__, __FILE__);
 }
 
+TEST(NVFuserTest, FusionMMASwizzlePrimitiveFloatAcc_CUDA) {
```

Comment:
Maybe we may want to put this to a new test file (e.g., "test_gpu_mma.cpp")

Reply:
Yes. I will also need to disable this test for <Volta archs.